pkg/kubelet/nodeshutdown/nodeshutdown_manager_linux.go

```diff
@@ -316,6 +316,7 @@ func (m *managerImpl) processShutdownEvent() error {
 				klog.V(1).InfoS("Shutdown manager killing pod with gracePeriod", "pod", klog.KObj(pod), "gracePeriod", gracePeriodOverride)
 
 				if err := m.killPodFunc(pod, false, &gracePeriodOverride, func(status *v1.PodStatus) {
+					status.Phase = v1.PodFailed
```

Comment:
needs the conditional check from master applied

Reply:
Done